.release-please-manifest.json

```diff
@@ -2,6 +2,7 @@
   "packages/calcite-components": "3.1.0",
   "packages/calcite-components-react": "3.1.0",
   "packages/calcite-design-tokens": "3.0.1",
+  "packages/calcite-tailwind-preset": "1.0.0",
```

Comment:
`release-please` will read this as the current version being `1.0.0`. Did you already publish v1 manually? If not you can remove this line and release-please will add the version in its PR.